File: exs/mundo_1/javascript/031.js
/*
Desafio 031

Problema: Desenvolva um programa que pergunte a distância de
          uma viagem em Km. Calcule o preço da passagem, cobrando
          R$0,50 por Km para viagens de até 200Km e R$0,45 para
          viagens mais longas.

Resolução do problema:
*/

const input = require('readline-sync').question //Chamando o modulo readline-sync para entrada de dados

//Entrada de dados
const dist = input("Qual distância percorrida?: ")

if(dist <= 200){
    let price = (dist * 0.50)
    console.log(`Preço da passagem: R$${price.toFixed(2)}`)
}
else {
    let price = (dist*0.45)
    console.log(`Preço da passagem: R$ ${price.toFixed(2)}`)
}

File: exs/mundo_1/golang/031.go
/*
Desafio 031

Problema: Desenvolva um programa que pergunte a distância de
          uma viagem em Km. Calcule o preço da passagem, cobrando
          R$0,50 por Km para viagens de até 200Km e R$0,45 para
          viagens mais longas.

Resolução do problema:
*/

package main

import(
	"fmt"
)

var d, preco float64

func main(){
	fmt.Print("Digite a distância em KM: ")
	fmt.Scanln(&d)

	if d > 200{
		preco = d * 0.45
	} else {
		preco = d * 0.50
	}

	fmt.Printf("Para %vKM, o preço fica R$%v.\n", d, preco)
}
